x-pack/plugins/cases/public/containers/use_get_tags.test.tsx

```diff
@@ -22,7 +23,9 @@ describe('useGetTags', () => {
 
   it('init', async () => {
     await act(async () => {
-      const { result, waitForNextUpdate } = renderHook<string, UseGetTags>(() => useGetTags());
+      const { result, waitForNextUpdate } = renderHook<string, UseGetTags>(() =>
+        useGetTags([SECURITY_SOLUTION_OWNER])
```

Comment:
I see that passing owner as an argument makes it much easier to test as you don't have to wrap it in a provider. 🤔